pyqtgraph/graphicsItems/TextItem.py

```diff
@@ -208,7 +208,7 @@ def updateTransform(self, force=False):
         angle = -self.angle
         if self.rotateAxis is not None:
             d = pt.map(self.rotateAxis) - pt.map(Point(0, 0))
-            a = np.arctan2(d.y(), d.x()) * 180 / np.pi
+            a = degrees(atan2(d.y(), d.x()))
             angle += a
         t.rotate(angle)  
```

Comment:
`rotateRadians()` instead?
Needs conversion of a, so it doesn't simplify things a lot though.

Reply:
I'm ok with this use-case; the `self.rotateAxis is not None` is a less likely case than not I figure, so I an see the case where that's the code-path that should take the (slight) hit in performance.

Reply:
Ah, it seems that it is self.angle that would need conversion here, and that seems to be defined in degrees. So sticking with plain `.rotate()` is probably best here.